app/models/miq_schedule.rb

```diff
@@ -331,10 +331,11 @@ def validate_file_depot  # TODO: Do we need this if the validations are on the F
   end
 
   def verify_file_depot(params)  # TODO: This logic belongs in the UI, not sure where
-    depot_class = FileDepot.supported_protocols[params[:uri_prefix]]
-    depot       = file_depot.class.name == depot_class ? file_depot : build_file_depot(:type => depot_class)
-    depot.name  = params[:name]
-    depot.uri   = params[:uri]
+    depot_class      = FileDepot.supported_protocols[params[:uri_prefix]]
+    depot            = file_depot.class.name == depot_class ? file_depot : build_file_depot(:type => depot_class)
+    depot.name       = params[:name]
+    depot.uri        = params[:uri]
+    # depot.aws_region = params[:aws_region]
```

Comment:
@jerryk55 should this be uncommented?

Reply:
Doh!  Absolutely.  I tested without the line prior to committing and obviously forgot to add it back in.  Thanks @carbonin.